data/gsettings/meson.build

```diff
@@ -0,0 +1,7 @@
+configure_file(
+  input: 'org.gnome.GPaste.gschema.xml.in',
+  configuration: conf,
+  output: 'org.gnome.GPaste.gschema.xml',
+  install: true,
+  install_dir: join_paths(get_option('datadir'), 'glib-2.0', 'schemas'),
+)
```

Comment:
We're losing the translation here, aren't we?

Reply:
Oh, you're doing both the translation and setting the conf variables in one step with autotools? It has been some time since I've done this, but IIRC there were no differences between the files autotools installed and my meson thingie installed. Please just GPaste with both of them into some local root and diff them for yourself :)

Reply:
I have to admit that I do not remember how translations work for the gsettings stuff